aws/terminator/data_services.py

```diff
@@ -199,6 +199,48 @@ def terminate(self):
         self.client.delete_db_instance(DBInstanceIdentifier=self.name, SkipFinalSnapshot=True)
 
 
+class RdsDbSnapshot(Terminator):
+    @staticmethod
+    def create(credentials):
+        return Terminator._create(credentials, RdsDbSnapshot, 'rds', lambda client: client.describe_db_snapshots()['DBSnapshot'])
+
+    @property
+    def id(self):
+        return self.instance['DBSnapshotArn']
+
+    @property
+    def name(self):
+        return self.instance['DBSnapshotIdentifier']
+
+    @property
+    def created_time(self):
+        return self.instance['SnapshotCreateTime']
```

Comment:
The `SnapshotCreateTime` key does not seem to always be present. It's working for some snapshots, but I'm getting a missing key error on at least one. I wonder if we should use `DbTerminator` for this one, instead? I can't find any information on why this key would be missing.